### DISCUSSION TITLE
Mac ce version display problem

### DISCUSSION THREAD

**Discussion by niudehua** [dbeaver/dbeaver#12045] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version   macOS 11.2.3
- DBeaver version       CE 21.0.2

#### Description ：
the system table  of information_schema appears in the view
![image](https://user-images.githubusercontent.com/36468944/113979267-9b5c9c00-9877-11eb-9965-5f447aae3a78.png)

**Comment by uslss** [comment, marked as answer]
> The INFORMATION_SCHEMA database contains several read-only tables. They are actually views, not base tables, so there are no files associated with them, and you cannot set triggers on them. 
https://dev.mysql.com/doc/refman/8.0/en/information-schema-introduction.html

So it is expected behavior in this case
